daprdocs/content/en/dotnet-sdk-docs/dotnet-actors/dotnet-actors-howto.md

```diff
@@ -219,6 +221,14 @@ namespace MyActorService
                 TimeSpan.FromSeconds(5));  // Time interval between reminder invocations after the first invocation
         }
 
+        /// <summary>
+        /// Get MyReminder reminder details with the actor
+        /// </summary>
+        public async Task<IActorReminder> GetReminder()
+        {
+            await this.GetReminderAsync("MyReminder");
+        }
```

Comment:
Personal preference, we should include this with an input string as an example. Otherwise people may think it can only take hardcoded names.

Reply:
I kept this as to be consistent with other APIs where the reminder-name is hardcoded in `RegisterReminder` and `UnregisterReminder`. Both of them take the reminder name as inputs as well, similar to `GetReminder`. 
https://github.com/dapr/dotnet-sdk/pull/1113/files/53876be8bf17e98bb9011094ab39623dc0be74e4#diff-763693d0f8815d098d3c9e4f284e8909335c17756bf549eef7abc550c3151a63L213
https://github.com/dapr/dotnet-sdk/pull/1113/files/53876be8bf17e98bb9011094ab39623dc0be74e4#diff-763693d0f8815d098d3c9e4f284e8909335c17756bf549eef7abc550c3151a63L225

It's also same as kept in examples: https://github.com/dapr/dotnet-sdk/blob/master/test/Dapr.E2E.Test.App/Actors/ReminderActor.cs

Reply:
Is this okay @halspang ?